docs/_custom/css/mkdoclingo.css

```diff
@@ -9,6 +15,19 @@ code.doc-clingo-symbol-predicate::after {
     color: #ca91bf
 }
 
+.md-grid {
+    max-width: 1440px;
+}
+
+.md-sidebar--secondary {
+    width: 15rem !important;
+    left: -15rem !important;
+}
+
+.md-nav--secondary {
+    width: 15rem !important;
+    left: -15rem !important;
+}
```

Comment:
This makes the sidebar larger to better fit the names

Reply:
I suggest we include this in the css of the project template ad add this as a suggestion in mkdoclingo docs